AllReadyApp/Web-App/AllReady/Services/TaskAttachmentService.cs

```diff
@@ -93,17 +88,6 @@ public AttachmentService(IOptions<AzureStorageSettings> options)
 
             Debug.WriteLine("Attachment uploaded to URI: " + blockBlob.Uri);
             return blockBlob.Uri.ToString();
-
-        }
-
-        public IList<string> GetAllowedExtensions()
-        {
-            return AllowedExtensions;
-        }
-
-        public int GetMaxAttachmentBytes()
-        {
-            return MaxAttachmentBytes;
         }
     }
 }
```

Comment:
although it's not in this changeset, can you please remove the `Option #1` comment as well as the `Option #2` comment along with the commented out code below it.